src/util/equalThoughtRanked.js

```diff
@@ -1,3 +1,3 @@
 /** Compares two thought objects using { value, rank } as identity and ignoring other properties. */
-export const equalThoughtRanked = (a, b) =>
-  a === b || (a && b && a.value === b.value && a.rank === b.rank)
+export const equalThoughtRanked = (a, b, isRankEqual = true) =>
```

Comment:
Rather than change the signature of this method, I would suggest simply comparing `a.value === b.value`.

Reply:
I did so because I wanted no other problems to occur in other parts that use this function.

Reply:
Right, this function should not be changed it all. The comparison `a.value === b.value` should be made at the site of the callee.

Reply:
@dezzzus